easylinks/src/main/java/com/google/step/servlets/EasyLinkServlet.java

```diff
@@ -0,0 +1,23 @@
+package com.google.step.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responses to redirect the easy link to another website*/
+@WebServlet("/easy-link")
+public class EasyLinkServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String easyLink = ServletHelper.getParameterWithDefault(request, "easy-link", );
+    if (!easyLink.startsWith(HTTP)) {
+      easyLink = HTTP + easyLink;
+    }
+    response.sendRedirect(easyLink);
+  }
+
+  private static final String HTTP = "http://";
```

Comment:
Not sure if this is right, but I guess most of the time we are using https://?

Reply:
We'll change it later when we finish implementing the datastore. After we finish the datastore, the user will type in the easy links, start with go/, ~where something like that. Now, it is there just to make sure we can redirect to the right page. For example, www.google.com won't work, we have to add 'http://' before it.